src/Twig/Runtime/NavbarExtensionRuntime.php

```diff
@@ -109,11 +109,26 @@ private function getCurrentRouteName(): string
 
     private function getActiveOptions(): array
     {
+        $options = [];
+
+        // don't use sortBy or time options on comment pages
+        // for the navbar links, so sorting comments by new does not mean
+        // changing the entry and microblog views to newest
+        if (!$this->isRouteName('root')
+            && !$this->isRouteNameStartsWith('front')
+            && !$this->isRouteNameStartsWith('posts')
+            && !$this->isRouteName('magazine_posts')
```

Comment:
unrelated but think we should rename this route to `posts_magazine` to match `front_magazine` that entry has, which matches all the other subcategories, that way this check could be removed and it would match closer to the entry routing